api/src/routes/user.rs

```diff
@@ -0,0 +1,175 @@
+//! Contains all user routes
+
+use config::{AdminToken, DbConn};
+use db::{self, models::User};
+use rocket::Route;
+use rocket::http::Status;
+use rocket::response::status::Custom;
+use rocket_contrib::Json;
+
+/// Returns all user routes
+pub fn get_routes() -> Vec<Route> {
+    routes![create, remove, get, get_all]
+}
+
+/**
```

Comment:
Meeeeh, I don't like mixing comment styles :'( 

Is there a way to use `///` instead of `/**`?

Reply:
No, apidoc doesn't support rust comment

Reply:
That sucks :(